Buildings/Templates/Plants/Controls/StagingRotation/EquipmentEnable.mo

```diff
@@ -233,6 +250,15 @@ equation
     annotation (Line(points={{-78,80},{-70,80},{-70,0},{-62,0}},color={0,0,127}));
   connect(voiStaZer.y, isReq.u)
     annotation (Line(points={{-78,80},{-70,80},{-70,-40},{-62,-40}},color={0,0,127}));
+  connect(cha.y, swiEna.u[1]) annotation (Line(points={{52,-60},{66,-60},{66,-60},
+          {78,-60}},                color={255,0,255}));
+  connect(intLes.y, swiEna.u[2]) annotation (Line(points={{52,-100},{62,-100},{
+          62,-60},{78,-60}}, color={255,0,255}));
+  connect(u1HeaCoo, cha1.u) annotation (Line(points={{-220,-120},{-180,-120},{
+          -180,-100},{-142,-100}}, color={255,0,255}));
+  connect(cha1.y, swiEna.u[3]) annotation (Line(points={{-118,-100},{-40,-100},{
```

Comment:
The input `swiEna.u[3]` will connect to nothing if `is_pumApp` is `true` (the block `cha1` will be disabled).